iModelCore/ECPresentation/Source/Hierarchies/NavNodesCacheHelpers.cpp

```diff
@@ -224,15 +226,15 @@ static NavNodeKeyPtr CreateNodeKeyFromStatement(Statement& stmt, IConnectionCR c
             return ECClassInstanceKey(connection.GetECDb().Schemas().GetClass(k.GetClassId()), k.GetInstanceId());
             });
         auto nodeKey = ECInstancesNodeKey::Create(classInstanceKeys, specificationIdentifier, hashPath);
-        SetInstanceKeysSelectQueryFromStatement(stmt, nodeKey);
+        SetInstanceKeysSelectQueryFromStatement(stmt, *nodeKey.get());

```

Comment:
You can get a reference from `RefCountedPtr` by using `*` -  no need to first get a raw pointer from it with `.get()`.

Reply:
Removed `.get()`.